src/plugins/dataflow/components/dataflow-tile.tsx

```diff
@@ -157,17 +163,24 @@ export default class DataflowToolComponent extends BaseComponent<IProps, IDatafl
 
   private pairNodesToAttributes = () => {
     const model = this.getContent();
+    // dataSet looks like
+    // Time_Quantized | Time_Actual | Node 1 | Node 2 | Node 3 etc
+    //    0           |  0          | val    | val    |  val
+    model.addNewAttrFromNode(0, "Time_Quantized");
+    model.addNewAttrFromNode(1, "Time_Actual");
```

Comment:
These columns don't correspond to nodes, so `addNewAttrFromNode()` seems inappropriate. You can just add them to the `DataSet`:
```suggestion
    addAttributeToDataSet(model.dataSet, { name: "Time_Quantized" });
    addAttributeToDataSet(model.dataSet, { name: "Time_Actual" });
```